media/js/base/mozilla-client.js

```diff
@@ -185,13 +185,13 @@ if (typeof Mozilla === 'undefined') {
         isESR = isESR === undefined ? false : isESR;
         userVer = userVer === undefined ? Client._getFirefoxVersion() : userVer;
 
-        var $html = $(document.documentElement);
+        var html = document.documentElement;
```

Comment:
🚀 👍

Reply:
I'll leave this improvement in even though it's not necessary anymore ;)